src/components/definition-list/dd.component.js

```diff
@@ -3,13 +3,11 @@ import propTypes from "@styled-system/prop-types";
 import PropTypes from "prop-types";
 import { StyledDd } from "./definition-list.style";
 
-const Dd = ({ mb = 2, children, ...props }) => {
-  return (
-    <StyledDd mb={mb} {...props}>
-      {children}
-    </StyledDd>
-  );
-};
+const Dd = ({ children, ...rest }) => (
+  <StyledDd mb={rest.mb || 2} {...rest}>
```

Comment:
That line looks a little strange. It should be defined that default `mb` is 2.

Reply:
The issue here is that if we define/ destructure the prop in the args we end up with some weird prop tables. Adding them as a default prop would work in the short term but eventually we will have to move all default props into the args again as that is the pattern react are moving towards. I'd prefer to destructure in the component and instead to avoid that, I don't think it's an issue as the default value is defined in the prop tables now I've updated it to use the `StyledSystemProps` table

![image](https://user-images.githubusercontent.com/44157880/131468087-590206d0-8113-487d-b5b4-2d6bbeee4f39.png)